tests/system/src/test/java/test/javafx/scene/text/TestHandlingBigGlyphID_Win10.java

```diff
@@ -0,0 +1,38 @@
+import javafx.application.Application;
+import static javafx.application.Application.launch;
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+import javafx.stage.Stage;
+
+public class TestHandlingBigGlyphID_Win10 extends Application {
```

Comment:
In order for this test to be run, you need to do the following:

1. The name of the test class must end with exactly `Test` -- so I suggest something like `BigGlyphIDTest` (no need for "Win10" to be in the name).

2. The test class itself cannot extend Application. We have plenty of examples of how this can be done in other tests.

3. In order to be an automated test, you need some sort of assertion checking.

Reply:
I today checked other tests codes.
TextTest.java and FontTest.java say
"StubFontLoader only knows about Amble."

So We can not load font "Arial Unicode MS" ?
If so, It looks impossible to create automated unit test code.

Reply:
As far as I know, "Arial Unicode MS"  is best font for this tests.
I tried to load font several way in automated test, but I could not success.
Is it OK to modify these test codes and move it to tests/manual/text/ ?